mlrun/datastore/__init__.py

```diff
@@ -22,6 +22,14 @@
     "NoSqlTarget",
     "StreamTarget",
     "KafkaTarget",
+    "ParquetStoreyTarget",
+    "CSVStoreyTarget",
+    "NoSqlStoreyTarget",
+    "StreamStoreyTarget",
+    "KafkaStoreyTarget",
+    "RedisNoSqlStoreyTarget",
+    "TSDBStoreyTarget",
+    "TDEngineStoreyTarget",
```

Comment:
Do we need to export this? It seems like it's meant for internal use to me.

Reply:
I believe we do, since we use 'mlrun.datastore.StreamStoreyTarget' for example when we add step to graph

Reply:
I'm pretty sure this is going to pollute the API that's visible to the user (most noticeably, the docs).